langchain/src/vectorstores/supabase.ts

```diff
@@ -79,9 +79,9 @@ export class SupabaseVectorStore extends VectorStore {
     const chunkSize = 500;
     let returnedIds: string[] = [];
     for (let i = 0; i < rows.length; i += chunkSize) {
-      const chunk = rows.slice(i, i + chunkSize).map((row) => {
+      const chunk = rows.slice(i, i + chunkSize).map((row, j) => {
         if (options?.ids) {
-          return { id: options.ids[i], ...row };
+          return { id: options.ids[j], ...row };
```

Comment:
Wait, should this be `i + j`?

Reply:
I equal 0 always 

example here : https://codesandbox.io/s/youthful-shirley-843zf9?file=/src/app.ts
refresh the browser to console log i and chunk